### PR TITLE
Temporarely set normalize_y=False to avoid divition by zero in sklear…

### LINKED ISSUE #5: Normalize_y=False to avoid problem with n_initial_points being asked
As in https://github.com/scikit-optimize/scikit-optimize/issues/947 ...
...and in https://github.com/scikit-learn/scikit-learn/issues/18371

... we risk getting an error message that is difficult to understans and hence to fix. Problem relates to a divition with zero when applying constantLiar method to ask for several experiments, when the current knowledge is zero (or one) point(s).

I suggest that we just change "normalize_y" to False (as preset value) in cook_estimator:
https://github.com/novonordisk-research/ProcessOptimizer/blob/8675ca0c7f4181f39cae3ba862faa51cb277a0cf/ProcessOptimizer/utils.py#L354

### PR DESCRIPTION
**Pull request by sqbl**
normalize_y set to False to avoid divide by zero error in SKLEARN > 0.23

### CODE REVIEW

**Comment by sqbl** [comment]
Merge, but please remember #8 to reset to previous value after changes in SKLEARN
